src/main/java/com/hivemq/cli/commands/MqttCommand.java

```diff
@@ -50,45 +52,55 @@
     private String identifierPrefix;
 
     public void setDefaultOptions() {
+        final DefaultCLIProperties defaultCLIProperties = MqttCLIMain.MQTTCLI.defaultCLIProperties();
```

Comment:
since DefaultCLIProperties is a Singleton, I suspect you could do that with dependency injection

Reply:
This is actually a small workaround because the properties would have to be injected into every command's constructor (which all extend `MqttCommand`) to be usable here. Therefore I get the current properties singleton from the injection in the main method.